models/falcon-40b-instruct.json

```diff
@@ -0,0 +1,86 @@
+{
+  "_descriptorVersion": "0.0.1",  
+  "datePublished": "2023-10-22T03:04:42",
+  "name": "falcon-40b-instruct",
+  "description": "Falcon-40B-Instruct, based on Falcon-40B, has been fine-tuned on chat and instruct datasets. It offers outstanding performance and is an excellent choice for chat and instruct applications. Falcon-7B-Instruct is part of the Falcon family of language models, known for their exceptional capabilities and openness.",
+  "author": {
+    "name": "Maddes8cht",
+    "url": "https://huggingface.co/maddes8cht",
+    "blurb": "Maddes8cht Passionate about Open Source and AI. On Hugging Face he is advocating for real open source AI models with OSI compliant licenses"
+  },
+  "numParameters": "40B",
```

Comment:
let's actually remove the 40B model from here: people might not realize the resources this model will require and they won't understand why it's not working. As a rule of thumb, catalog models should be 13B and below